### PR TITLE
[4.0] Language reinstall

### PR DESCRIPTION
**Pull request by brianteeman**
Adds a visual indicator to the button for reinstalling a language by changing the button color

PR for #30808

image

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on d154fd0e96d0a21f44bef7bddbea767744b0cb30

thanks @brianteeman<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30810">issues.joomla.org/tracker/joomla-cms/30810</a>.</sub>

File: administrator/components/com_installer/tmpl/languages/default.php [hunk 1]
```diff
@@ -67,8 +67,9 @@
 							<tr class="row<?php echo $i % 2; ?>">
 								<td>
 									<?php $buttonText = (isset($this->installedLang[0][$language->code]) || isset($this->installedLang[1][$language->code])) ? 'REINSTALL' : 'INSTALL'; ?>
+									<?php $buttonClass = (isset($this->installedLang[0][$language->code]) || isset($this->installedLang[1][$language->code])) ? 'btn btn-success btn-sm' : 'btn btn-primary btn-sm'; ?>
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/languages/default.php:70]:
```suggestion
									<?php $buttonClass = $buttonText === 'REINSTALL' ? 'btn btn-success btn-sm' : 'btn btn-primary btn-sm'; ?>
```

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on d154fd0e96d0a21f44bef7bddbea767744b0cb30<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30810">issues.joomla.org/tracker/joomla-cms/30810</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30810">issues.joomla.org/tracker/joomla-cms/30810</a>.</sub>

**Comment by HLeithner** [comment]
thanks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**